dask_glm/utils.py

```diff
@@ -3,6 +3,7 @@
 import inspect
 import sys
 
+import dask.distributed as dd
```

Comment:
`dd` is the usual alias for `dask.dataframe`. Could you change this to an `import distributed` or `from distributed import Client?

```suggestion
from distributed import Client
```

Reply:
Agree I will do. 
I did the change with actors and ran unit tests on it. It is very simple to use. I would like to do a full run on the cluster here to validate my change before to push the commit.